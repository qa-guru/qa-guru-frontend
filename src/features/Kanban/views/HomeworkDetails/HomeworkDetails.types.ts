import { StudentHomeWorkDto } from "../../../../api/graphql/generated/graphql";

export interface IHomeworkDetail {
  card: StudentHomeWorkDto;
  onClose: () => void;
  showHomeworkDetails: boolean;
}
