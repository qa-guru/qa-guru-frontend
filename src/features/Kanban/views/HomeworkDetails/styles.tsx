export const style = {
  menu: {
    padding: 3,
    maxHeight: "76vh",
    overflowY: "auto",
  },
  paper: {
    padding: {
      xs: "15px",
      md: "20px",
    },
    marginTop: "30px",
  },
};
