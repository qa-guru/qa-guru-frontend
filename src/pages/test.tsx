import { FC } from "react";
import { useParams } from "react-router-dom";

import TestContainer from "features/test/containers/test-container";

const TestPage: FC = () => {
  const { testId, trainingId, lectureId } = useParams<{
    testId: string;
    trainingId: string;
    lectureId: string;
  }>();

  // Проверяем, что все параметры присутствуют
  if (!testId || !trainingId || !lectureId) {
    return <div>Ошибка: отсутствуют необходимые параметры</div>;
  }

  return (
    <TestContainer
      testId={testId}
      trainingId={trainingId}
      lectureId={lectureId}
    />
  );
};

export default TestPage;
