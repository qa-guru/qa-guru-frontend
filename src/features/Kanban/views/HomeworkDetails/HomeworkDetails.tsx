import React from "react";
import { Box, Button, Paper, Stack, Typography } from "@mui/material";
import ChevronRightIcon from "@mui/icons-material/ChevronRight";
import { format, parseISO } from "date-fns";
import { style } from "./styles";
import { IHomeworkDetail } from "./HomeworkDetails.types";
import { getFormattedId } from "../../helpers/getFormattedId";
import UserRow from "../../../../shared/components/UserRow";
import { ReactComponent as MentorIcon } from "../../../../assets/icons/mentor.svg";
import { ReactComponent as StudentIcon } from "../../../../assets/icons/student.svg";
import StatusContent from "../../../../shared/components/StatusContent";
import TextSerialization from "../../../../shared/Serializers/TextSerialization";
import LectureHomework from "../../../../shared/components/LectureHomework";

const HomeworkDetails: React.FC<IHomeworkDetail> = ({ card, onClose }) => {
  const Format = "dd.MM.yyyy | HH:mm";

  return (
    <Box sx={style.menu}>
      <Stack spacing={1} direction="row" justifyContent={"space-between"}>
        <Typography variant="h6">{getFormattedId(card.id!)}</Typography>
        <Button onClick={onClose} variant="contained">
          Свернуть <ChevronRightIcon />
        </Button>
      </Stack>
      <Typography variant="body1" mt="2vh">
        {card.lecture?.subject}
      </Typography>
      <Stack spacing={2} direction="row" mt="15px">
        <UserRow
          icon={StudentIcon}
          user={card.student!}
          width={26}
          height={26}
        />
        {card.mentor && (
          <UserRow
            icon={MentorIcon}
            user={card.mentor}
            width={26}
            height={26}
          />
        )}
      </Stack>
      <Stack spacing={2} direction="row" mt={2}>
        <Stack direction="column">
          <Typography variant="body2">Создано</Typography>
          <Typography variant="caption" noWrap>
            {card.creationDate && format(parseISO(card.creationDate), Format)}
          </Typography>
        </Stack>
        {card.startCheckingDate && (
          <Stack direction="column">
            <Typography variant="body2">Начало проверки</Typography>
            <Typography variant="caption" noWrap>
              {card.startCheckingDate &&
                format(parseISO(card.startCheckingDate), Format)}
            </Typography>
          </Stack>
        )}
        {card.endCheckingDate && (
          <Stack direction="column">
            <Typography variant="body2">Окончание проверки</Typography>
            <Typography variant="caption">
              {card.endCheckingDate &&
                format(parseISO(card.endCheckingDate), Format)}
            </Typography>
          </Stack>
        )}
      </Stack>
      <Box mt="15px">
        <StatusContent status={card.status!} />
      </Box>
      <LectureHomework lectureHomeWork={card.lecture?.contentHomeWork!} />
      <Paper sx={style.paper}>
        <Typography variant="h5" mb={2}>
          Ответ на задание
        </Typography>
        <TextSerialization text={card.answer!} />
      </Paper>
    </Box>
  );
};

export default HomeworkDetails;
