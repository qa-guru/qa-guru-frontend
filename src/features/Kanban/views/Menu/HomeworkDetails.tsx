import React from "react";
import { Box, Button, Grid, Typography } from "@mui/material";
import ChevronRightIcon from "@mui/icons-material/ChevronRight";
import { style } from "./styles";
import { IHomeworkDetail } from "./HomeworkDetails.types";
import { getFormattedId } from "../../helpers/getFormattedId";

const HomeworkDetails: React.FC<IHomeworkDetail> = ({ card, onClose }) => {
  return (
    <Box sx={style.menu}>
      <Grid container direction={"row"} justifyContent={"space-between"}>
        <Grid item>
          <Typography variant="h6">{getFormattedId(card.id!)}</Typography>
        </Grid>
        <Grid item>
          <Button onClick={onClose} variant="contained">
            Свернуть <ChevronRightIcon />
          </Button>
        </Grid>
      </Grid>
    </Box>
  );
};

export default HomeworkDetails;
