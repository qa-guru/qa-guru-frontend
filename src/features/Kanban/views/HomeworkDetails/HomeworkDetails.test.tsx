import { MemoryRouter } from "react-router-dom";
import HomeworkDetails from "./HomeworkDetails";
import { render } from "../../../../test/utilsTest";
import { studentHomeWorkDto } from "../../../../shared/mocks/studentHomeWorkDto.mock";

describe("HomeworkDetails", () => {
  it("renders correctly", () => {
    const { asFragment } = render(
      <MemoryRouter>
        <HomeworkDetails
          card={studentHomeWorkDto}
          onClose={() => {}}
          showHomeworkDetails={true}
        />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it("renders correctly", () => {
    const { asFragment } = render(
      <MemoryRouter>
        <HomeworkDetails
          card={studentHomeWorkDto}
          onClose={() => {}}
          showHomeworkDetails={false}
        />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
