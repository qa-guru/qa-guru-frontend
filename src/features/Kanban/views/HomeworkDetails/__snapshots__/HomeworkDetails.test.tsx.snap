// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeworkDetails > renders correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
    style="opacity: 1; webkit-transition: opacity 700ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 700ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  >
    <svg
      fill="none"
      height="24"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20 20V4H22V20H20ZM2 20L2 4H4L4 20H2ZM9 11V8L5 12L9 16V13H15V16L19 12L15 8V11H9Z"
        fill="#2CCCA6"
      />
    </svg>
    <div
      class="MuiBox-root css-m71bw6"
    >
      <div
        class="MuiStack-root css-jibmk2-MuiStack-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
        >
          JAVA-23
        </h6>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Свернуть 
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="ChevronRightIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
      <p
        class="MuiTypography-root MuiTypography-body1 css-ix1f03-MuiTypography-root"
      />
      <div
        class="MuiStack-root css-9v4aab-MuiStack-root"
      >
        <div
          class="MuiStack-root css-unhrs1-MuiStack-root"
        >
          <svg
            fill="none"
            height="16"
            viewBox="0 0 20 16"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4.16683 8.98333V12.3167L10.0002 15.5L15.8335 12.3167V8.98333L10.0002 12.1667L4.16683 8.98333ZM10.0002 0.5L0.833496 5.5L10.0002 10.5L17.5002 6.40833V12.1667H19.1668V5.5L10.0002 0.5Z"
              fill="#2CCCA6"
            />
          </svg>
          <div
            class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-4ljh0t-MuiAvatar-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
            >
              fl
            </h6>
          </div>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-10wpov9-MuiTypography-root"
            >
              firstName lastName
            </h6>
          </div>
        </div>
        <div
          class="MuiStack-root css-unhrs1-MuiStack-root"
        >
          <svg
            fill="none"
            height="20"
            viewBox="0 0 20 20"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M17.5002 2.5H2.50016C1.5835 2.5 0.833496 3.25 0.833496 4.16667V10.8333H2.50016V4.16667H17.5002V17.5C18.4168 17.5 19.1668 16.75 19.1668 15.8333V4.16667C19.1668 3.25 18.4168 2.5 17.5002 2.5Z"
              fill="#2CCCA6"
            />
            <path
              d="M7.49984 11.6667C9.34079 11.6667 10.8332 10.1743 10.8332 8.33333C10.8332 6.49238 9.34079 5 7.49984 5C5.65889 5 4.1665 6.49238 4.1665 8.33333C4.1665 10.1743 5.65889 11.6667 7.49984 11.6667Z"
              fill="#2CCCA6"
            />
            <path
              d="M12.8252 13.8C11.4252 13.0833 9.6085 12.5 7.50016 12.5C5.39183 12.5 3.57516 13.0833 2.17516 13.8C1.34183 14.225 0.833496 15.0833 0.833496 16.0167V18.3333H14.1668V16.0167C14.1668 15.0833 13.6585 14.225 12.8252 13.8Z"
              fill="#2CCCA6"
            />
          </svg>
          <div
            class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-4ljh0t-MuiAvatar-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
            >
              fl
            </h6>
          </div>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-10wpov9-MuiTypography-root"
            >
              firstName lastName
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiStack-root css-9v4aab-MuiStack-root"
      >
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            Создано
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption MuiTypography-noWrap css-g98qyn-MuiTypography-root"
          >
            10.10.2021 | 13:10
          </span>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            Начало проверки
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption MuiTypography-noWrap css-g98qyn-MuiTypography-root"
          >
            10.10.2021 | 15:10
          </span>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            Окончание проверки
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
          >
            10.10.2021 | 13:11
          </span>
        </div>
      </div>
      <div
        class="MuiBox-root css-tr0r3x"
      >
        <div
          class="MuiStack-root css-1q36rmd-MuiStack-root"
        >
          <svg
            fill="none"
            height="20"
            viewBox="0 0 20 20"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.99 0C4.47 0 0 4.48 0 10C0 15.52 4.47 20 9.99 20C15.52 20 20 15.52 20 10C20 4.48 15.52 0 9.99 0ZM10 18C5.58 18 2 14.42 2 10C2 5.58 5.58 2 10 2C14.42 2 18 5.58 18 10C18 14.42 14.42 18 10 18Z"
              fill="black"
              fill-opacity="0.56"
            />
            <path
              d="M10.5 5H9V11L14.25 14.15L15 12.92L10.5 10.25V5Z"
              fill="black"
              fill-opacity="0.56"
            />
          </svg>
          <div
            class="MuiBox-root css-0"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
            >
              Ожидает проверки
            </p>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-16cch88-MuiPaper-root"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 css-ch2k2f-MuiTypography-root"
        >
          Домашнее задание
        </h5>
        <div
          class="MuiStack-root css-zgbp5c-MuiStack-root"
        />
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-6qxqkz-MuiPaper-root"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 css-ch2k2f-MuiTypography-root"
        >
          Ответ на задание
        </h5>
        <div
          class="MuiBox-root css-i6bazn"
        >
          <h6
            class="MuiTypography-root MuiTypography-subtitle1 css-18u75fl-MuiTypography-root"
          >
            answer
            <br />
          </h6>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`HomeworkDetails > renders correctly 2`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
    style="opacity: 0; visibility: hidden;"
  >
    <svg
      fill="none"
      height="24"
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20 20V4H22V20H20ZM2 20L2 4H4L4 20H2ZM9 11V8L5 12L9 16V13H15V16L19 12L15 8V11H9Z"
        fill="#2CCCA6"
      />
    </svg>
    <div
      class="MuiBox-root css-m71bw6"
    >
      <div
        class="MuiStack-root css-jibmk2-MuiStack-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
        >
          JAVA-23
        </h6>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          Свернуть 
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="ChevronRightIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
      <p
        class="MuiTypography-root MuiTypography-body1 css-ix1f03-MuiTypography-root"
      />
      <div
        class="MuiStack-root css-9v4aab-MuiStack-root"
      >
        <div
          class="MuiStack-root css-unhrs1-MuiStack-root"
        >
          <svg
            fill="none"
            height="16"
            viewBox="0 0 20 16"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4.16683 8.98333V12.3167L10.0002 15.5L15.8335 12.3167V8.98333L10.0002 12.1667L4.16683 8.98333ZM10.0002 0.5L0.833496 5.5L10.0002 10.5L17.5002 6.40833V12.1667H19.1668V5.5L10.0002 0.5Z"
              fill="#2CCCA6"
            />
          </svg>
          <div
            class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-4ljh0t-MuiAvatar-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
            >
              fl
            </h6>
          </div>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-10wpov9-MuiTypography-root"
            >
              firstName lastName
            </h6>
          </div>
        </div>
        <div
          class="MuiStack-root css-unhrs1-MuiStack-root"
        >
          <svg
            fill="none"
            height="20"
            viewBox="0 0 20 20"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M17.5002 2.5H2.50016C1.5835 2.5 0.833496 3.25 0.833496 4.16667V10.8333H2.50016V4.16667H17.5002V17.5C18.4168 17.5 19.1668 16.75 19.1668 15.8333V4.16667C19.1668 3.25 18.4168 2.5 17.5002 2.5Z"
              fill="#2CCCA6"
            />
            <path
              d="M7.49984 11.6667C9.34079 11.6667 10.8332 10.1743 10.8332 8.33333C10.8332 6.49238 9.34079 5 7.49984 5C5.65889 5 4.1665 6.49238 4.1665 8.33333C4.1665 10.1743 5.65889 11.6667 7.49984 11.6667Z"
              fill="#2CCCA6"
            />
            <path
              d="M12.8252 13.8C11.4252 13.0833 9.6085 12.5 7.50016 12.5C5.39183 12.5 3.57516 13.0833 2.17516 13.8C1.34183 14.225 0.833496 15.0833 0.833496 16.0167V18.3333H14.1668V16.0167C14.1668 15.0833 13.6585 14.225 12.8252 13.8Z"
              fill="#2CCCA6"
            />
          </svg>
          <div
            class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-4ljh0t-MuiAvatar-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 css-ati914-MuiTypography-root"
            >
              fl
            </h6>
          </div>
          <div
            class="MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-10wpov9-MuiTypography-root"
            >
              firstName lastName
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiStack-root css-9v4aab-MuiStack-root"
      >
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            Создано
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption MuiTypography-noWrap css-g98qyn-MuiTypography-root"
          >
            10.10.2021 | 13:10
          </span>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            Начало проверки
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption MuiTypography-noWrap css-g98qyn-MuiTypography-root"
          >
            10.10.2021 | 15:10
          </span>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            Окончание проверки
          </p>
          <span
            class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
          >
            10.10.2021 | 13:11
          </span>
        </div>
      </div>
      <div
        class="MuiBox-root css-tr0r3x"
      >
        <div
          class="MuiStack-root css-1q36rmd-MuiStack-root"
        >
          <svg
            fill="none"
            height="20"
            viewBox="0 0 20 20"
            width="20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.99 0C4.47 0 0 4.48 0 10C0 15.52 4.47 20 9.99 20C15.52 20 20 15.52 20 10C20 4.48 15.52 0 9.99 0ZM10 18C5.58 18 2 14.42 2 10C2 5.58 5.58 2 10 2C14.42 2 18 5.58 18 10C18 14.42 14.42 18 10 18Z"
              fill="black"
              fill-opacity="0.56"
            />
            <path
              d="M10.5 5H9V11L14.25 14.15L15 12.92L10.5 10.25V5Z"
              fill="black"
              fill-opacity="0.56"
            />
          </svg>
          <div
            class="MuiBox-root css-0"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
            >
              Ожидает проверки
            </p>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-16cch88-MuiPaper-root"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 css-ch2k2f-MuiTypography-root"
        >
          Домашнее задание
        </h5>
        <div
          class="MuiStack-root css-zgbp5c-MuiStack-root"
        />
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-6qxqkz-MuiPaper-root"
      >
        <h5
          class="MuiTypography-root MuiTypography-h5 css-ch2k2f-MuiTypography-root"
        >
          Ответ на задание
        </h5>
        <div
          class="MuiBox-root css-i6bazn"
        >
          <h6
            class="MuiTypography-root MuiTypography-subtitle1 css-18u75fl-MuiTypography-root"
          >
            answer
            <br />
          </h6>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
